### DISCUSSION TITLE
How to get form data in custom validator?

### DISCUSSION THREAD

**Discussion by l-yk-l** [filamentphp/filament#3868] [Help] [answered]
I need to get an attribute value in custom rule for other attribute. How can I do this?

**Comment by danharrin** [comment, marked as answer]
I don't think Laravel supports that in custom rule callbacks yet?

**Reply by ibnnajjaar** [reply]
In Laravel you can pass value of an attirbute to a custom rule by doing something like this.

```php 
'start_at' => 'date',
'end_at' => ['date', new IsValidSlotTime($this->input('start_at')] 
```
Is there a way you can pass other attribute values like that in filament?

**Reply by ibnnajjaar** [reply]
Found a way to pass values of other attributes to a custom validation rule by doing as follows.

```php

->rules([
        function (callable $get) {
          $start_at = $get("start_at");
          return new IsValidSlotStartTime($start_at);
        },])

```

**Reply by l-yk-l** [reply]
Oh, thank you so much!
